data-source/json/test_list_change_evaluates_sections.json

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
This schema is far larger than it needs to be, it should be the minimum required to test the feature.